source/Playnite.Common.NetStandard/ProcessStarter.cs

```diff
@@ -58,6 +58,42 @@ public static Process StartProcess(string path, string arguments, string workDir
             }
 
             return Process.Start(info);
-        }        
+        }
+
+        /// <summary>
+        /// Starts a process by using the cmd, this avoids the game to be started as child of the process that is executing it
+        /// </summary>
+        /// <param name="path"></param>
+        /// <param name="arguments"></param>
+        /// <param name="workDir"></param>
+        /// <param name="asAdmin"></param>
+        /// <returns></returns>
+        public static Process StartProcessWithCMD(string path, string arguments, string workDir, bool asAdmin = false)
+        {
+            logger.Debug($"Starting process with cmd: {path}, {arguments}, {workDir}");
+            var startupPath = path;
+            if (path.Contains(".."))
+            {
+                startupPath = Path.GetFullPath(path);
+            }
+
+            var directory = Path.GetDirectoryName(startupPath);
+            var gameExe = Path.GetFileName(startupPath);
+
+            var info = new ProcessStartInfo
+            {
+                FileName = "cmd",
+                Arguments = $"/C start {gameExe} {arguments}",
+                WorkingDirectory = string.IsNullOrEmpty(workDir) ? directory : workDir,
+                WindowStyle = ProcessWindowStyle.Hidden
+            };
+
+            if (asAdmin)
```

Comment:
why is this necessary?

Reply:
The API of playnite has an argument asAdmin, if you want to launch the game as administrator if is installed on a path that has admin permissions, normally this shouldn't be necessary, and it is called always as asAdmin = false, but I mantain this option.